proposal/README.md

```diff
@@ -1 +1,94 @@
-proposal/requirement
+SUSE SolidDriver Checks
+=======================
+
+The purpose of the SUSE SolidDriver Checks is to ensure best end user
+experience by verifying that third party delivered drivers are built and
+deployed in a SUSE SolidDriver compliant] manner.
+
+General Features
+----------------
+*NOTE: Not all features are implemented at this time*
+
+* This tool shall be usable by SUSE, SUSE Partners, and SUSE customers.
+* The tool shall be able check running and installed modules on local or
+  remote systems
+* The tool shall be able check kernel module packages (KMPs) that have not
+  been installed
+* The tool shall create reports in the following formats:
+    - HTML
+    - Excel spreadsheet
+    - PDF
+    - JSON
+    - Terminal
+* Remote checks executed via ssh preferably using ssh keys for
+  authentication
+
+
+Checks
+------
+*NOTE: Not all checks are implemented at this time*
+
+* Critical checks failures shall give a high level warning (RED)
+* Important checks failures shall give a low level warning (AMBER)
+
+
+### Kernel Modules
+
+Critical checks
+
+* Kernel modules have `supported` flag properly set
+    - `supported` flag appears only once
+    - `supported` flag value exactly matches the string `external`
```

Comment:
```no``` for ```kernel-default-extra-***``` packages is also acceptable.

Reply:
Even though it's intentional we need to create the same warning since those modules are unsupported. Customers need to be aware of that.

I was thinking that we should simply ignore modules from official SUSE packages since this tool should be focused on checking third party modules, but in the case of kernel-defaul-extra, it is good to report that unsupported modules are installed or maybe even used.